src/mixins/map-mixin.js

```diff
@@ -545,12 +538,17 @@ export default function mapMixin(
         coordinates: boundsToUse
       })
 
+      const mapLayers = _chart.getAllLayers()
+      const chartType =
+        Array.isArray(mapLayers) && mapLayers[0] && mapLayers[0].type
+      const opacity = chartType === "heatmap" ? 0.5 : 0.85
```

Comment:
@CaitW, is that mean we can't go higher than 50% of opacity for the the heatmap and 85% for all the other charts?
We have an opacity adjuster slider in Master layer view and that takes the current opacity value as it is but will allow to increase the opacity to full 100%. Though, with this change, it sets the 50% and 85% to the full 100%, so we can't go up but just lower. Please check the attached image for clarification.
<img width="1656" alt="Screen Shot 2019-08-12 at 10 58 21 AM" src="https://user-images.githubusercontent.com/11098472/62886592-422e1500-bcf0-11e9-992e-d589ad16f369.png">
<img width="1651" alt="Screen Shot 2019-08-12 at 11 00 41 AM" src="https://user-images.githubusercontent.com/11098472/62886764-a94bc980-bcf0-11e9-89b0-990ea12750a2.png">

Reply:
Yep - you're right. I didn't think about multi-layer maps when I hard-coded the values. I'll add a fix.